src/playwright/test.ts

```diff
@@ -126,17 +274,17 @@ export const test = baseTest.extend<{
   addEventListener: (
     locator: Locator,
     event: string,
-    options?: {
+    options: {
```

Comment:
`options` is now required because the sole purpose of `addEventListener()` is now to cancel and stop propagation of events.